androidsdk/src/main/java/com/wootric/androidsdk/Wootric.java

```diff
@@ -83,85 +118,216 @@ public void setEndUserEmail(String email) {
      */
     public void setEndUserCreatedAt(long createdAt) {
 
-        if (createdAt > 9999999999L){
-            Log.v(TAG, "WARNING: The created date exceeds the maximum 10 characters allowed. " +
+        if (createdAt > 9999999999L) {
```

Comment:
I changed this so we check if the date is on the future. If it is, we let the user know through a log. Is this ok or should I do some additional checks?